### DISCUSSION TITLE
SpiralLinux 11.220610 release

### DISCUSSION THREAD

**Discussion by geckolinux** [SpiralLinux/SpiralLinux-project#11] [Announcements]
![SpiralLinux-transparent-small](https://user-images.githubusercontent.com/2692138/172185018-77411974-5e2b-43b6-aed8-42f492047cd6.png)

### https://spirallinux.github.io

The SpiralLinux project is pleased to announce the 11.220610 release with a few fixes and improvements on top of the initial [11.220606 release](https://github.com/SpiralLinux/SpiralLinux-project/releases/tag/11.220606).

## Changes

- Added `spice-vdagent` for automatic screen resizing and clipboard integration with the host system when running as a libvirt guest (Gnome Boxes, etc.) 
- Plasma edition now includes the `upower` package to fix the bug where batteries weren't being detected (thanks to [Akram Abunahla](http://disq.us/p/2pb56af) for reporting)
- All editions now include a fix for an issue that was sometimes preventing the bootloader from installing correctly in Calamares manual partitioning mode (thanks to [samarkv](https://github.com/SpiralLinux/SpiralLinux-project/issues/7) for reporting)

## About SpiralLinux

SpiralLinux is a selection of Linux spins built from Debian GNU/Linux, with a focus on simplicity and out-of-the-box usability across all the major desktop environments. SpiralLinux serves as an alternative live installation method for a highly reliable customized Debian system using only official Debian package repositories.

### What will happen to GeckoLinux?

GeckoLinux will continue to be maintained. However, SUSE and openSUSE have announced some major changes that will eventually affect the Leap branch a few years from now, so SpiralLinux would fill the void in the event that openSUSE Leap ceases to exist or turns into a completely different sort of product.

<hr /><em>This discussion was created from the release <a href='https://github.com/SpiralLinux/SpiralLinux-project/releases/tag/'>SpiralLinux 11.220610 release</a>.</em>

**Comment by bittin** [comment]
Gonna try SpiralLinux on my laptop today :)
